### DISCUSSION TITLE
Why the latest version is not v3.4.4 but v3.3.10?

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [argoproj/argo-workflows#10278] [Q&A] [answered]
Hello, I have a question.
Why the latest version is not v3.4.4 but v3.3.10?

v3.4 has already been released and GA.
But the latest version is v3.3.10.

<img width="192" alt="image" src="https://user-images.githubusercontent.com/13323303/209463213-2180a127-c061-4226-a8ff-bbe6acf665a2.png">

<img width="221" alt="image" src="https://user-images.githubusercontent.com/13323303/209463219-8dc86bca-a14d-4836-b4db-7cd64dc02529.png">

Is there any reason v3.4 isn't latest?

**Comment by terrytangyuan** [comment, marked as answer]
Thanks. It should be fixed now.
